sass/components/_media_iframe.scss

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file
+ * Media Iframe specific styles.
+ */
+.ecl-media-container--custom-ratio {
+  display: inline-block;
+
+  .ecl-media-container__media--ratio-custom {
+    display: flex;
+  }
+
+  .ecl-media-container__caption {
+    display: flex;
+
+    > div {
```

Comment:
Maybe add some documentation to explain why we add this line.

Reply:
Added